RefactoringsExperiments/src/main/java/ru/hse/kirilenko/refactorings/extractors/PosExtractionRunner.java

```diff
@@ -11,25 +7,21 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class ExtractionRunner {
+public class PosExtractionRunner {
```

Comment:
`pos` means positive?

Reply:
Was my best idea on how to differentiate positive and negative extractions

Reply:
`PositiveExtractionRunner` would be better